src/components/RichRadio/RichRadio.tsx

```diff
@@ -112,7 +115,14 @@ export const RichRadio = ({
     <StyledRoot className={className} onValueChange={onChange} value={value}>
       <VStack spacing="space-half">
         {items.map(item => (
-          <StyledItem value={item.value} key={item.value}>
+          <StyledItem
+            value={item.value}
+            data-tip={item.tooltip}
+            key={item.value}
+          >
+            {item.tooltip && (
+              <ReactTooltip {...{ effect: 'solid', ...item.tooltipSetting }} />
```

Comment:
What do you think about using this syntax for prop spreading?
```suggestion
              <ReactTooltip effect="solid" {...item.tooltipSetting } />
```

Reply:
That definitely looks better than mine 😄 Thank you